src/components/typography/typography.css

```
@@ -0,0 +1,33 @@
+h1,
+h2,
+h3,
+h5,
+h6 {
+  font-family: var(--ff-primary);
+}
+
+h4 {
+  font-family: var(--ff-secondary);
+  font-size: var(--h4-fs);
+  font-weight: var(--h4-fw);
+}
+
+h2 {
+  font-size: var(--h2-fs);
+  font-weight: var(--h2-fw);
+}
+
+h3 {
+  font-size: var(--h3-fs);
+  font-weight: var(--h3-fw);
+}
+
+h5 {
+  font-size: var(--h5-fs);
+  font-weight: var(--h5-fw);
+}
+
+h6 {
+  font-size: var(--h6-fs);
+  font-weight: var(--h6-fw);
+}
```

Here I'd use the variables you already declared and the `rem` values directly.

```suggestion
h2 {
  font-size: var(--h2-fs);
  font-weight: var(--h2-fw);
}

h3 {
  font-size: var(--h3-fs);
  font-weight: var(--h3-fw);
}

h4 {
  font-family: var(--ff-secondary);
  font-size: var(--h4-fs);
  font-weight: var(--h4-fw);
}

h5 {
  font-size: var(--h5-fs);
  font-weight: var(--h5-fw);
}

h6 {
  font-size: var(--h6-fs);
  font-weight: var(--h6-fw);
}
```

---

h5 und h6 kann ich löschen, da sie nirgendwo im Template gebraucht werden oder?